lib/nodes/Func.js

```diff
@@ -12,7 +12,7 @@ const { registerWalker } = require('../walker');
 
 const Container = require('./Container');
 
-const colorFunctions = ['hsl', 'hsla', 'rgb', 'rgba'];
+const colorFunctionsLowerRegex = /^(hsla?|hwb|lab|lch|rgba?)$/;
```

Comment:
```suggestion
const reColorFunctions = /^(hsla?|hwb|lab|lch|rgba?)$/i;
```

It looks like units are case insensitive https://www.w3.org/TR/css-values-3/#dimensions, so we can modify the regex accordingly and also do away with the `lowerName` on line 106 if we want to.

Reply:
Done in https://github.com/shellscape/postcss-values-parser/pull/111/commits/b95c4c8bcdd541be6aa93da16ac0e51ab5188549